src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -19,7 +19,8 @@ class NUnit extends ConventionTask {
     def timeout
     def runList
     def run
-    boolean useX86 = false
+	String customReportFolder    
```

Comment:
Dear Francois,

I haven’t seen the wrong indent. I have to check it again.

Regards,
Marco

From: Francois Valdy [mailto:notifications@github.com]
Sent: Thursday, November 05, 2015 3:31 PM
To: Ullink/gradle-nunit-plugin <gradle-nunit-plugin@noreply.github.com>
Cc: Marco Zwyssig <marco@zwyssig.info>
Subject: Re: [gradle-nunit-plugin] support for setting report directory (#10)


In src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy<https://github.com/Ullink/gradle-nunit-plugin/pull/10#discussion_r44016932>:

> @@ -19,7 +19,8 @@ class NUnit extends ConventionTask {

>      def timeout

>      def runList

>      def run

> -    boolean useX86 = false

> +     String customReportFolder

indent

—
Reply to this email directly or view it on GitHub<https://github.com/Ullink/gradle-nunit-plugin/pull/10/files#r44016932>.